index.js

```diff
@@ -31,14 +31,16 @@ let rcl = {
   _nodes: [],
 
   createNode(nodeName, namespace = '') {
-    let handle = rclnodejs.createNode(nodeName, namespace);
+    if (typeof (nodeName) !== 'string' || typeof (namespace) !== 'string') {
+      throw new Error('Wrong type of argments.');
```

Comment:
Consider using TypeError: "The TypeError object represents an error when a value is not of the expected type" from:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError